vector/src/main/java/im/vector/app/features/home/room/detail/composer/VoiceMessageHelper.kt

```diff
@@ -60,6 +60,7 @@ class VoiceMessageHelper @Inject constructor(
         try {
             voiceRecorder.startRecord()
         } catch (failure: Throwable) {
+            Timber.e(failure, "Unable to start recording")
```

Comment:
do these logs end up in the rageshakes?

Reply:
Yes, the Tree is planted [here](https://github.com/vector-im/element-android/blob/develop/vector/src/main/java/im/vector/app/VectorApplication.kt#L133)

Reply:
good to know, thanks!